src/index.ts

```diff
@@ -159,9 +159,9 @@ function resolveLaravelPlugin(pluginConfig: Required<PluginConfig>): LaravelPlug
                     fs.writeFileSync(hotFile, viteDevServerUrl)
 
                     setTimeout(() => {
-                        server.config.logger.info(colors.red(`\n  Laravel ${laravelVersion()} `))
-                        server.config.logger.info(`\n  > APP_URL: ` + colors.cyan(appUrl))
-                    })
+                        server.config.logger.info(colors.red(`\n  ${colors.bold('Laravel')} ${laravelVersion()} `))
+                        server.config.logger.info(`  ${colors.green('➜')}  ${colors.bold('APP_URL')}: ${colors.cyan(appUrl.replace(/:(\d+)/, (_, port) => `:${colors.bold(port)}`))}`)
+                    }, 100)
```

Comment:
I hate that I'm specifying a timeout here, however I was unable to get it to work without something here and was unable to find any additional hooks